src/Serilog.Sinks.BrowserHttp/Sinks/BrowserHttp/BrowserHttpSink.cs

```diff
@@ -60,6 +61,15 @@ class BrowserHttpSink : PeriodicBatchingSink
 			_httpClient = messageHandler == null ?
 				new HttpClient() { } :
 				new HttpClient(messageHandler) { };
+
+			if(httpHeaders != null)
+            {
+				var keys = httpHeaders.Keys;
+				foreach(var key in keys)
+                {
+					_httpClient.DefaultRequestHeaders.Add(key, httpHeaders[key]);
+                }
+            }
```

Comment:
Looks like this file has ended up with a mix of space and tab - should be all spaces (not sure where those tabs crept in 😅 ) .. anything consistent would be fine, though.

The indexing back in by key isn't super idiomatic with `IDictionary<,>`; maybe we could:

```csharp
foreach (var header in httpHeaders)
{
    _httpClient.DefaultRequestHeaders.Add(header.Key, header.Value)
}
```